noether_simulator/package.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<package>
```

Comment:
The rest of noether is package.xml format 2. We should convert this to 2 as well.